cpp/ql/src/semmle/code/cpp/commons/Scanf.qll

```diff
@@ -223,35 +195,36 @@ class ScanfFormatLiteral extends Expr {
     //                 2 - maximum width
     //                 3 - length modifier
     //                 4 - conversion character
-    result = "(\\%("+this.getMaxWidthRegexp()+")("+this.getLengthRegexp()+")("+this.getConvCharRegexp()+")).*"
+    result = "(\\%(" + this.getMaxWidthRegexp() + ")(" + this.getLengthRegexp() + ")(" +
+        this.getConvCharRegexp() + ")).*"
   }
 
   /**
    * Holds if the arguments are a parsing of a conversion specifier to this format string.
-   *  @param n     which conversion specifier to parse
-   *  @param spec  the whole conversion specifier
-   *  @param width the maximum width option of this input (empty string if none is given)
-   *  @param len   the length flag of this input (empty string if none is given)
-   *  @param conv  the conversion character of this input
+   *
+   * @param n     which conversion specifier to parse
```

Comment:
This has gone badly wrong.

Reply:
It turns out `@param` doesn't actually work in QLDoc and never has. I'll add a separate commit to address this and two other instances where it's used.